_posts/2023-10-26-superuser.md

```diff
@@ -0,0 +1,118 @@
+---
+layout: post
+title: "[CVE-2023-45899] Improper Access Control in superuser module edited by idnovate for PrestaShop"
+categories: modules
+author:
+- 202-ecommerce.com
+- TouchWeb.fr
+- Friends-Of-Presta.org
+meta: "CVE,PrestaShop,superuser"
+severity: "high (7.5)"
+---
+
+The module "idnovate" for PrestaShop restrict or incorrectly restricts access to the "connect as" feature from >= 2.3.5 and < 2.4.2 lets an attacker connect as any customer account. Release 2.4.2 fixed this security issue.
```

Comment:
restrict -> restricts
Par contre je ne comprends pas la phrase "restricts or incorrectly restricts". Ce n'est pas simplement "incorrectly restricts" ?

lets an -> which lets an

Reply:
La source de mon texte que j'ai mal recopié d'ailleurs
https://cwe.mitre.org/data/definitions/284.html

Reply:
Oui il te manque le `does not` alors :)
> does not restrict